framework/tests/spotify_ui_test.py

```diff
@@ -25,7 +25,8 @@ class TestSpotifyUI:
     def test_artist_song(self, artist, song, browser):
         self.__spotify_home_form.click_search_btn()
         self.__spotify_home_form.input_search_field(artist)
-        assert self.__artist_form.is_song_exists(song), f"{artist}'s {song} was not found"
+        with allure.step("Asserting if expected song exists in artists"):
+            assert self.__artist_form.is_song_exists(song), f"{artist}'s {song} was not found"
 
         allure.attach(browser.driver.get_screenshot_as_png(), name=f"{artist}_{song}_screenshot",
```

Comment:
let's attach screenshot in fixture instead, to have a screenshot in case of assertion failure